src/Core/tree_policies/UCT_TreePolicy.cpp

```diff
@@ -9,7 +10,8 @@ std::shared_ptr<SearchNode> UCT_TreePolicy::treePolicy(std::shared_ptr<SearchNod
             return m_expand(current_node);
         }
 
-        current_node = m_bestChild(current_node);
+        double cp = 1 / std::sqrt(2);
```

Comment:
```suggestion
        constexpr sqrt2 = std::sqrt(2);
        double cp = 1 / sqrt2;
```
Slight performance increase

Reply:
constexpr is computed at compile time, and sqrt is a relatively expensive operation

Reply:
Coolio